dbt/models/sec10k/core_sec10k__quarterly_company_information/schema.yml

```diff
@@ -8,10 +8,24 @@ sources:
               table_name: core_sec10k__quarterly_company_information
               partition_column: report_date
         columns:
```

Comment:
I think I'm supposed to update these `dbt` schema files, but I'm unfamiliar with all the dbt stuff so not sure I did this right.

Reply:
Oh sorry, my initial comment was phrased poorly. I meant - I updated these schema files, but I wasn't sure if I was actually supposed to. 

Seems like the rest of the comment is directed at @zschira and is more about formatting and diffs. Could be moved to a separate issue?